cudamapper/src/cudamapper_utils.cpp

```diff
@@ -24,6 +24,8 @@
 #include <claraparabricks/genomeworks/utils/cudautils.hpp>
 #include <claraparabricks/genomeworks/utils/signed_integer_utils.hpp>
 
+#include <claraparabricks/genomeworks/cudamapper/utils.hpp>
```

Comment:
I think it would be better to do this include this where needed (`main.cu`)

Reply:
Fixed, I think. Included only in source file (utils.cpp), main.cu, and sample_cudamapper.cpp

Reply:
Ok. As mentioned in the general comment, I would create a separate `.cpp` file for this header

Reply:
I might be misunderstanding something, but I created a new `.cpp` file in `cudamapper/src/utils.cpp` that corresponds to the `utils.hpp` header. Is this what you meant?

Reply:
yes, looks good now!